### PR TITLE
fix(ujust): Fix faulty ujust install-resolve

### PR DESCRIPTION
**Pull request by Zeglius**

### CODE REVIEW

File: system_files/desktop/shared/usr/share/ublue-os/just/82-bazzite-apps.just [hunk 1]
```diff
@@ -213,9 +215,10 @@ install-resolve ACTION="":
           echo "${red}Please place the file DaVinci_Resolve_*_Linux.run in ${HOME} or ${DOWNLOADDIR}${n}";
           exit 1
         fi
-        $SETUPSCRIPT $RUNFILE
-        echo "${b}Instalation was successful${n}"
-        echo "${dim}TIP: You can always update by running 'ujust install-resolve'${n}"
+        cmdline="env -C $tmpdir $SETUPSCRIPT $RUNFILE && rm -rf ${tmpdir}"
+        echo -e "${b}To finish install, run this command in a terminal:${n}\n\n${cmdline}\n\n"
+        wl-copy "${cmdline}"
+        echo "Command was set in clipboard"
```
Comment (tristann21) [inline system_files/desktop/shared/usr/share/ublue-os/just/82-bazzite-apps.just:221]:
```suggestion
        echo "Command was copied to clipboard"
```

**Review by KyleGospo — APPROVED** [review]

**Comment by Zeglius** [comment]
Renaming this PR from "fix(ujust): Avoid selinux conflicts in ujust install-resolve" to "fix(ujust): Fix faulty ujust install-resolve".

The reason of the initial name was result of the assumption it was caused by SELinux (which journalctl didn't log given my paranoia with the former).

Turns out is an oddity with Appimages and the current working dir of the bash session (which by default is /usr/share/ublue-os).

TL;DR: This doesnt work:
```
a.AppImage --appimage-extract
```
This does:
```
bash <<< "env -C $tmpdir a.AppImage --appimage-extract"
```

**Review by nicknamenamenick — APPROVED** [review]
